### PR TITLE
Edited readme and docs directory; added TM&B

### PR DESCRIPTION
**Pull request by DougTW**
Signed-off-by: DougTW <doug.martin@intel.com>

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Quick-start
 
-The following describes the Minimal steps to get started with CRI Resource
-Manager.
+The following describes the minimum number of steps to get started with CRI
+Resource mManager.
```
Comment (klihub) [inline docs/quick-start.md:4]: typo: 'mManager'

Comment (DougTW) [reply docs/quick-start.md]: Thanks for catching that typo! Latest commit fixed it.

**Comment by klihub** [comment]
@DougTW Could you rebase this on the latest master. We need to get rid of that test build failure for merging, and the fix for that has been just committed and merged to master.

**Comment by klihub** [comment]
@DougTW Also, there are a few more things I spotted:
- trailing whitespace in a few of the files (node-agent.md, quick-start.md, setup.md), touched
- a few files withour newline at EOF (README.md, contributing.md, node-agent.md)

**Comment by klihub** [comment]
Closing. Merged the same branch with commit history squashed into a single commit.

**Pull request closed without merging**